deployment/provision_vagrant_vm.sh

```diff
@@ -11,9 +22,13 @@ export DEBIAN_FRONTEND=noninteractive
 
 # install python stuff
 apt-get -q update
-apt-get -q install -y python3.7 python3.7-dev python3-venv python3.7-venv gettext
+apt-get -q install -y sudo python3.7 python3.7-dev python3-venv python3.7-venv gettext git wget npm
```

Comment:
Why did you add `npm`? We install node and npm using `nvm` at the end of this file